From f98edc687f6f6ff430c6da81ac8772d8aa6f60f9 Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Wed, 9 Jan 2013 16:21:28 -0800
Subject: [PATCH] power: Add POWER_HINT_APP_PROFILES to powerhal

This change adds POWER_HINT_APP_PROFILES to list of hints
available to powerhal.

Bug 1155948

Change-Id: Ibe11110882490e19aa8eb9ad4258d29cf4b457ae
Reviewed-on: http://git-master/r/191129
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Prajakta Gudadhe <pgudadhe@nvidia.com>
---
 include/hardware/power.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index ee266f3..193ba68 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -41,6 +41,7 @@ __BEGIN_DECLS
 typedef enum {
     POWER_HINT_VSYNC = 0x00000001,
     POWER_HINT_INTERACTION = 0x00000002,
+    POWER_HINT_APP_PROFILE = 0x00000003,
     POWER_HINT_COUNT,
 } power_hint_t;
 
-- 
1.8.1.5

